py/legacypipe/runbrick.py

```diff
@@ -2611,6 +2661,9 @@ def stage_writecat(
 
     # Brick pixel positions
     ok,bx,by = targetwcs.radec2pixelxy(T2.orig_ra, T2.orig_dec)
+    # iterative sources
+    bx[ok==False] = 1.
+    by[ok==False] = 1.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comparison to False should be 'if cond is False:' or 'if not cond:'](https://app.codacy.com/app/dstndstn/legacypipe/pullRequest?prid=4730106)